contracts/AutomatedFunctionsConsumer.sol

```diff
@@ -86,8 +89,23 @@ contract AutomatedFunctionsConsumer is FunctionsClient, ConfirmedOwner, Automati
     s_lastUpkeepTimeStamp = block.timestamp;
     s_upkeepCounter = s_upkeepCounter + 1;
 
-    bytes32 requestId = _sendRequest(s_requestCBOR, s_subscriptionId, s_fulfillGasLimit, donId);
-    s_lastRequestId = requestId;
+    try
+      i_router.sendRequest(
```

Comment:
Why not _sendRequest any more?

Reply:
`try-catch` can only be used for external function calls

Reply:
TIL, thanks!